experiments/kripke/experiment.py

```diff
@@ -39,102 +39,72 @@ class Kripke(
 
     maintainers("pearce8")
 
-    def compute_applications_section(self):
-        # TODO: Replace with conflicts clause
-        scaling_modes = {
-            "strong": self.spec.satisfies("+strong"),
-            "weak": self.spec.satisfies("+weak"),
-            "throughput": self.spec.satisfies("+throughput"),
-            "single_node": self.spec.satisfies("+single_node"),
-        }
-
-        scaling_mode_enabled = [key for key, value in scaling_modes.items() if value]
-        if len(scaling_mode_enabled) != 1:
-            raise BenchparkError(
-                f"Only one type of scaling per experiment is allowed for application package {self.name}"
-            )
-
-        input_variables = {
+    def initialize_expr_variables(self):
```

Comment:
kripke: global size
strong scaling: always 64 x 64 x 64
weak scaling: 2 procs 64 x 64 x 64 -> on 4 process is 64 x 64 x 128 -> on 8 is 64 x 128 x 128